file/tests/test.sh

```diff
@@ -1,21 +1,18 @@
 #!/bin/sh
 
+set -euo pipefail
+
 TESTDIR="$(dirname "${0}")"
-PLANDIR="$(dirname "${TESTDIR}")"
-SKIPBUILD=${SKIPBUILD:-0}
 
-hab pkg install --binlink core/bats
+if [ -z "${1:-}" ]; then
+  echo "Usage: $0 FULLY_QUALIFIED_PACKAGE_IDENT"
+  exit 1
+fi
```

Comment:
You might want to adjust this test.sh to be in line with the way it was done in the [sed pr](https://github.com/habitat-sh/core-plans/pull/2755/files)

Reply:
The way the usage text is handled? I don't think it matters.